tools/java/org/unicode/cldr/test/XPathToMenu.java

```diff
@@ -98,24 +101,13 @@ public static String xpathToMenu(String path) {
         // }
     }
 
-    public static String[] getMetazonesItems() {
-        String defaultMetazonesItems = "Africa America Antarctica Asia Australia Europe Atlantic Indian Pacific";
-        return (defaultMetazonesItems.split(" "));
+    private static String getMetazoneContinent(String xpath) {
+        XPathParts parts = XPathParts.getFrozenInstance(xpath);
+        String thisMetazone = parts.getAttributeValue(3, "type");
+        return getMetazoneToContinentMap().get(thisMetazone);
     }
 
-    private static Map<String, String> mzXpathToContinent = new HashMap<>();
```

Comment:
This was all kinds of weird since the map is never written to so was always empty.
It'd have more thoughts on this if I wasn't able to just delete the whole method.